presto-parquet/src/main/java/com/facebook/presto/parquet/predicate/TupleDomainParquetPredicate.java

```diff
@@ -200,6 +201,15 @@ else if (isVarcharType(type) && statistics instanceof BinaryStatistics) {
             ParquetStringStatistics parquetStringStatistics = new ParquetStringStatistics(minSlice, maxSlice);
             return createDomain(type, hasNullValue, parquetStringStatistics);
         }
+        else if (type.equals(DATE) && statistics instanceof IntStatistics) {
+            IntStatistics intStatistics = (IntStatistics) statistics;
+            // ignore corrupted statistics
+            if (intStatistics.genericGetMin() > intStatistics.genericGetMax()) {
+                return Domain.create(ValueSet.all(type), hasNullValue);
```

Comment:
@nezihyigitbasi do you know the reason why we silently ignore something? I see we do this already for other cases above.

Reply:
I don't exactly remember the history behind that. @zhenxiao do you remember why we are silently ignoring corrupt stats?

Reply:
as far as I remembered, if the statistics is obviously incorrect/corrupted(in this case, min > max), we silently do not leverage statistics to evaluate predicates.
it should not error out, as if stats are incorrect/corrupted, we could still scan data to complete the query, just lost a potential speedup.
maybe adding a log message here? @findepi @nezihyigitbasi what do you think?

Reply:
In general I prefer failing fast & loudly for such cases as that may uncover bugs/issues in whoever has written those files/stats.

Reply:
get it. fail loudly is better. I will file following task to fix it

Reply:
here it is: https://github.com/prestodb/presto/pull/12036